blame.php

```diff
@@ -81,7 +81,8 @@
 
 $vars['rev'] = $rev;
 $vars['peg'] = $peg;
-$vars['path'] = escape($ppath);
+$vars['path'] = str_replace('%2F', '/', rawurlencode($ppath));
```

Comment:
This looks confusing to me: I would have expected `path` to be a plain path, not an encoded URL. What's the reason for doing so? Is `path` always used in some URL-context? Wouldn't it be better to distinguish the plain path from an encoded URL using different variable names, something like `urlPath` or `pathAsUrl` or alike? Pretty much like `safepath` currently.

Reply:
I can explain that. If you look at the tempates: `websvn:path` is solely used iwth `href` attributes which require URI encoding. Paths which appear as text are prefixed with `safe` since they aren't usable in `href` attributes. Thhe code did not properly address this and assumed that files would be 7 bit ASCII only.

Reply:
So how about renaming `path` to something mentioning it to be a URL instead? Too risky/too much work or simple global search&replace? `safepath` seems badly named as well and is more like a `htmlPath`.